tasks/init.json

```diff
@@ -0,0 +1,51 @@
+{
+  "description": "Make a HTTP or HTTPS request.",
```

Comment:
It'd be good to specify input_method stdin here, if only to exemplify best practices (and also because we don't use env vars)